tensorboard/uploader/uploader_main.py

```diff
@@ -530,12 +530,16 @@ def execute(self, server_info, channel):
                 ("Id", experiment.experiment_id),
                 ("Created", util.format_time(experiment.create_time)),
                 ("Updated", util.format_time(experiment.update_time)),
-                ("Scalars", str(experiment.num_scalars)),
                 ("Runs", str(experiment.num_runs)),
                 ("Tags", str(experiment.num_tags)),
+                ("Scalars", str(experiment.num_scalars)),
             ]
+            if experiment.total_blob_bytes:
+                data.append(
+                    ("Binary object bytes", str(experiment.total_blob_bytes))
```

Comment:
Need to update fieldmask above to actually request `total_blob_bytes`?

Reply:
Done.